app/models/user.rb

```diff
@@ -1,12 +1,20 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Good job on adding columns that are responsible for the confirmation process to your schema, Kindly add the `:confirmable` attribute which is responsible for the confirmation process to the user model as requested by the previous reviewer. so it will be like:
```ruby
	devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```